src/main/java/decide/DecideHelpFunctions.java

```diff
@@ -1,5 +1,6 @@
 package decide;
-import javafx.geometry.Point2D;
+
+import java.awt.geom.Point2D;
```

Comment:
From what i tested, this version of Point2D doesn't support the angle() function, which might be useful in a few of the LIC functions